packages/data/src/checkParameter.ts

```diff
@@ -1,8 +1,11 @@
 /**
- * Check if the parameter type is correct.
- * @param value Parameter value.
- * @param name Parameter name.
- * @param type Expected parameter type.
+ * Validates the type of a given parameter against an expected type.
+ * Throws a {@link TypeError} if the validation fails.
+ * This function is useful for ensuring that function arguments conform to expected types at runtime.
+ * @param value The value of the parameter to check.
+ * @param name The name of the parameter, used in the error message for easier debugging.
+ * @param type The expected JavaScript type as a string (e.g., 'string', 'number', 'object').
+ * @throws {TypeError} Throws an error if the type of `value` does not match the `type`.
  */
 export default function checkParameter(value: any, name: string, type: string) {
```

Comment:
Not sure if this should be part of this PR, but we have a ZK-Kit utils package to check parameters now. We should use it instead of this function.

Reply:
hmmm I don't think so - we can open a small issue and amend it later.